readme.md

```diff
@@ -68,7 +70,18 @@ cp .env.example .env
 ```
 
 To generate an application key use: 
-`php artisan key:generate` this will add an application to your `.env` file. 
+`php artisan key:generate` this will add an application to your `.env` file.
+
+Create new `mysql` user and database
+```bash
+mysql -u root -h localhost -p
+mysql> CREATE USER 'username'@'localhost' IDENTIFIED BY 'password';
+mysql> GRANT ALL PRIVILEGES ON office_automation.* TO 'username'@'localhost' WITH GRANT OPTION;
+mysql> ALTER USER 'username'@'localhost' IDENTIFIED WITH mysql_native_password by 'password';
+mysql> \q
+mysql -u username -h localhost -p
+mysql> CREATE DATABASE office_automation;
+```
```

Comment:
This is very subjective, and user dependent.

Reply:
actually a lot of information in this readme is subjective and user dependent... like installation of prerequisites... Well, user can also use the `root` user here by modifying the `root` user's `IDENTIFIED WITH` as done here... 
would you like to propose some changes?

Reply:
My changes will also be very specific, because I handle `mysql-server` in a different way. Though I haven't tried running the project locally. But that's something which I don't prefer doing.

Again, it's all subjective so I won't propose any change. It's fine then.

Reply:
Yea its fine for now... Because if anyone  know what they have to do, they dont need to follow the complete readme... I'll merge this for now... we might write up readme from scratch...